cp1404practicals/prac_06/guitar.py

```diff
@@ -0,0 +1,26 @@
+"""
+Prac_06 - Daniel Keppie
+Guitar
+"""
+
+
+class Guitar:
+
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name.title()
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        return "{} ({}) : ${}".format(self.name, self.cost, self.year)
+
+    def get_age(self):
+        current_year = 2020
```

Comment:
current year could be a constant that way if you use this again and the year changes it will be much easier to change.